### DISCUSSION TITLE
rdk_doc/Advanced_development/toolchain_development/intermediate/ptq_process

### DISCUSSION THREAD

**Discussion by giscus** [D-Robotics/rdk_doc#5] [General]
# rdk_doc/Advanced_development/toolchain_development/intermediate/ptq_process

简介

http://10.64.29.52:1688/rdk_doc/Advanced_development/toolchain_development/intermediate/ptq_process

**Comment by xuguosheng110** [comment]
great！！！

**Comment by lukediag** [comment]
从头到尾把这章看完了，相当不错，内容详细，条例清晰

**Comment by ReduceDomain** [comment]
norm_type: 'data_mean_and_scale'
    mean_value: '103.94 116.78 123.68'
    scale_value: '0.017'
量化配置这样设置，
校准数据还需要做减均值和乘系数的操作吗？
推理数据还需要做减均值和乘系数操作吗？
